scripts/cover.sh

```diff
@@ -43,6 +43,7 @@ lcov --remove coverage.total \
     '*opae-libs/plugins/xfpga/usrclk/**' \
     '*tests/**' \
     '*tools/extra/c++utils/**' \
+    '*tools/libboard/board_a10gx/**' \
```

Comment:
are you not planning to test this one?

Reply:
RC card board module , we are not going to support RC card in next Release.

Reply:
Can we do this in a separate pull request?

Reply:
fixed, removed changes in merge request.